README.md

```diff
@@ -25,6 +27,47 @@ jobs:
       # ...
 ```
 
+
+### Using authn_id
+
+```yaml
+on: [push]
+
+jobs:
+  test:
+    # ...
+    permissions:
+      id-token: 'write'
+    steps:
+      # ...
+      - name: Import Secrets using CyberArk Conjur Secret Fetcher
+        uses: quincycheng/conjur-action@master
```

Comment:
This would need to be committed as `infamousjoeg/conjur-action@master` in order to fall in line with this repository. I believe this is a remnant of your testing prior to submitting PR.

Reply:
Sure. Updated.